### ISSUE TITLE
Date/Time mention on each task

### ISSUE THREAD

**Issue by mayurbhavsar45** [bappygolder/oTask#79]
Its will be good when we show date/time of each task when we create like 1 miniute ago , 1 hours ago, 5 sec , 02 jan 2016 : 12:00 AM

**Comment by bappygolder** [comment]
Good point Mayur. Currently oTask work as a daily todo list. So that after the end of the day you start with a new list again. So in that case do you think data/time will be a necessary feature?

However as we move forward and allow users to create multiple list data & time might be a useful feature then. 

I have users who requested a reminder option. However we need wait for that until we have integrated user account I think.
